### DISCUSSION TITLE
Multiple Nuki hub instances (for multiple locks) - locking one lock locks all others

### DISCUSSION THREAD

**Discussion by ohadbenita** [technyon/nuki_hub#466] [Q&A] [answered]
Hi, 

What could be the explanation of the following behavior:
- I have 3 Nuki hubs installed on 3 different ESP32 devices.
- Each of the hubs are connected to a separate Nuki lock and are exposed through HomeAssistant's auto-discovery mechanism.
- When attempting to lock/unlock one lock from HomeAssistant, all others are locked/unlocked with it.

Would appreciate any feedback about this.

**Comment by iranl** [comment]
The answer is in the FAQ

**Reply by ohadbenita** [reply, marked as answer]
Absolutely right, this was about the MQTT topic of the different hubs, thanks 👍
